build/dyn_shim_detection.py

```diff
@@ -31,5 +31,5 @@
 
 sys.path.append("{}/build/".format(spark_rapids_source_basedir))
 from get_buildvers import _get_buildvers
-value = _get_buildvers(buildvers, "{}/pom.xml".format(multi_module_project_dir), _log)
-project.setProperty("included_buildvers", value)
\ No newline at end of file
+value = _get_buildvers(buildvers, "{}/pom.xml".format(multi_module_project_dir), _log, isShimplify=True)
```

Comment:
It is hard to follow why this callsite is supposed to be  `isShimplify=True`. Should it rather refer to what this flag is affecting such as `ignore_excluded_shims=True` or `honor_excluded_shims=False` with a comment what the difference is.